debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/LogicalDecodingMessageMonitor.java

```diff
@@ -64,35 +63,24 @@ public class LogicalDecodingMessageMonitor {
     private final Schema blockSchema;
     private final Schema valueSchema;
 
+    private static final PostgresSchemaFactory postgresSchemaFactoryObject = PostgresSchemaFactory.get();
```

Comment:
@ani-sha Please remove this attribute

Reply:
@jpechane if we get rid of that how are we calling its related schemas in `PostgresSchemaFactory` ie `logicalDecodingMessageMonitorKeySchema` etc?

Reply:
@ani-sha Just inline the `get()` call, no need for variable.